debian/control

```diff
@@ -2,118 +2,117 @@ Source: ubuntu-system-settings
 Section: gnome
 Priority: optional
 Maintainer: Ubuntu Desktop Team <ubuntu-desktop@lists.ubuntu.com>
-Build-Depends: dbus-test-runner,
-               debhelper (>= 9),
-               dh-migrations,
-               dh-python,
-               python3-setuptools,
-               python3-all:any,
-               cmake,
-               gdb:any,
-               intltool,
-               libaccountsservice-dev,
-               libandroid-properties-dev [amd64 arm64 armhf i386],
-               libclick-0.4-dev,
-               libgeonames-dev,
-               libglib2.0-dev (>= 2.37.92),
-               libevdev-dev,
-               libicu-dev,
-               libpolkit-agent-1-dev,
-               libqmenumodel-dev,
-               libtrust-store-dev,
-               libudev-dev,
-               libunity-api-dev,
-               libupower-glib-dev,
-               pkg-config,
-               qml-module-qtcontacts,
-               qml-module-qtquick-layouts,
-               qml-module-qtsysteminfo (>= 5.0~),
-               qt5-default,
-               qtbase5-dev,
-               qtdeclarative5-dev,
-               qtdeclarative5-dev-tools,
-               qtdeclarative5-ubuntu-history0.1,
-               qtdeclarative5-ubuntu-keyboard-extensions0.1 [armhf arm64 amd64 i386],
-               qtdeclarative5-ubuntu-download-manager0.1 (>= 1.3~),
-               libapt-pkg-dev,
-               libgnome-desktop-3-dev,
-               libgsettings-qt-dev,
-# test-deps
-               dpkg-dev <!nocheck>,
-               qml-module-qttest <!nocheck>,
-               qml-module-qtquick2 <!nocheck>,
-               qml-module-ubuntu-components | qml-module-ubuntu-components-gles <!nocheck>,
-               qtbase5-private-dev <!nocheck>,
-               qtdeclarative5-ubuntu-settings-components (>= 0.8) <!nocheck>,
-               libqtdbusmock1-dev (>= 0.2+14.04.20140724) <!nocheck>,
-               libqtdbustest1-dev <!nocheck>,
-               libqt5sql5-sqlite <!nocheck>,
-               xvfb <!nocheck>,
-               pep8 <!nocheck>,
-               python3-pep8 <!nocheck>,
-               python3-flake8 (>= 2.2.2-1ubuntu4) | python3-flake8:native <!nocheck>,
-               python3:any <!nocheck>,
-               python3-gi <!nocheck>,
-               python3-dbus <!nocheck>,
-               python3-xdg <!nocheck>,
-               gir1.2-glib-2.0 <!nocheck>,
+Build-Depends:
```

Comment:
I'm not really a fan if this kind of "control file" structure.

Reply:
OK? Like some specific concern? Or what? It's less useless whitespace, organized alphabetically, etc… so it makes it easier to read and find issues.

Reply:
It was better with more whilespace to match the length of the category, but i guess it makes sense.